internal/controller/postgrescluster/controller.go

```diff
@@ -156,7 +155,7 @@ func (r *Reconciler) Reconcile(
 		primaryCertificate       *v1.SecretProjection
 		pgUser                   *v1.Secret
 		rootCA                   *pki.RootCertificateAuthority
-		monitoringSecret         *corev1.Secret
+		monitoringSecret         *v1.Secret
```

Comment:
📝 This change is fine. New code should use `corev1`.

Reply:
Updated to use `corev1`.